app/views/jobseekers/employments/_employments.html.slim

```diff
@@ -0,0 +1,21 @@
+- employments.each do |employment|
+  = render DetailComponent.new(title: employment.job_title, classes: ["work_history_entry"]) do |detail_component|
+    - detail_component.body do
+      = govuk_summary_list do |summary_list|
+        - summary_list.row(html_attributes: { id: "organisation" }) do |row|
```

Comment:
For these `id`s we'll end up with multiple elements on the same page with the same `id` - maybe interpolate with the index or something?

Reply:
the ids arent needed so removed alltogether